dpctl/tests/test_sycl_program.py

```diff
@@ -31,6 +31,24 @@ def get_spirv_abspath(fn):
     return spirv_file
 
 
+def _check_multi_kernel_program(prog):
+    assert type(prog) is dpctl_prog.SyclProgram
```

Comment:
Should we be using `assert`? (Recently concluded Security training speaking here)